static/app/views/dashboards/controls.tsx

```diff
@@ -178,6 +182,35 @@ function Controls({
                 {t('Export Dashboard')}
               </Button>
             </Feature>
+            <Feature features="dashboards-favourite">
+              <Button
+                size="sm"
+                aria-label={'dashboards-favourite'}
+                icon={
+                  <IconStar
+                    color={isFavorited ? 'yellow300' : 'subText'}
+                    isSolid={isFavorited}
+                    data-test-id={isFavorited ? 'yellow-star' : 'empty-star'}
```

Comment:
Instead of selecting it by the test ID, we can label this for accessibility with `aria-label={isFavorited ? t('Unfavorite') : t('Favorite')}` and then use `xByLabelText` in our tests to check the state